packages/protocol/test-sol/stability/SortedOracles.t.sol

```diff
@@ -80,6 +82,69 @@ contract SetReportExpiry is SortedOraclesTest {
   }
 }
 
+contract SortedOracles_SetEquivalentToken is SortedOraclesTest {
```

Comment:
This test would be a good candidate for fuzzing

Reply:
Added fuzzy tests